client/progress/show_progress.py

```
@@ -0,0 +1,59 @@
+            specials: Include specials as season 0 in progress
+            count_specials: Count specials in overall stats when specials included
+            last_activity: Calculate last/next episode based on 'aired' or 'watched'
+
+        Returns:
+            Show progress response with aired/completed counts and episode details
+
+        Raises:
+            ValueError: If not authenticated
+        """
+        if not self.is_authenticated():
+            raise ValueError("You must be authenticated to access show progress")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)